src/components/clusterConfiguration/HostsTable.tsx

```diff
@@ -59,7 +64,7 @@ const hostToHostTableRow = (openRows: OpenRows) => (host: Host, idx: number): IR
       cells: [
         id,
         {
-          title: <RoleDropdown role={role} host={host} />,
+          title: roleCellTitle,
```

Comment:
I would disable change via props within `RoleDropdown` component instead of changing it to static text

Reply:
Hmm, that would imply that there is a certain possibility to eventually change the role but after installation is started it is always fixed

Reply:
We can move responsibility for the whole cell down to the component. But ok, it's nitpicking.